test/chainrules.jl

```diff
@@ -13,6 +13,15 @@
         end
     end
 
+    for x in -2.0:2.0
+        test_frule(xexpx, x)
+        test_rrule(xexpx, x)
+        for y in -2.0:2.0
+            test_frule(xexpy, x, y)
+            test_rrule(xexpy, x, y)
```

Comment:
```suggestion
            test_scalar(xexpy, x, y)
```

Reply:
There is no `test_scalar(xexpy, x, y)` with multiple args :(